data/avgi/avgi ships.txt

```diff
@@ -0,0 +1,2378 @@
+	description "The Phononos can carry an impressive amount of cargo with the same standardized containers used by the Avlaki and Harmonikos, which can be quickly swapped out at the destination for a set of empty ones without needing to unload them."
+
+
+ship "Frequentia"
+	sprite "ship/avgi frequentia"
+	thumbnail "thumbnail/avgi frequentia"
+	attributes
+		category "Utility"
+		"cost" 1920000
+		"shields" 600
+		"hull"	1800
+		"required crew" 2
+		"bunks" 6
+		"mass" 180
+		"drag" 4.5
+		"heat dissipation" 1.2
+		"fuel capacity" 2400
+		"cargo space" 240
+		"outfit space" 186
+		"engine capacity" 60
+		"anchor point" 1
+		weapon
+			"blast radius" 360
+			"shield damage" 1800
+			"hull damage" 1200
+			"hit force" 2400
+	outfits
+		"Lantern Fission Core"
+		"Ultracapacitor Cell" 3
+		"Multispectral Scanner"
+		"Cryogenic Deuterium Tank" 6
+		"Electrostatic Ramscoop"
+		"Electromotive Rifle" 2
+
+		`Z-330 "Spark" Fusion Torch`
+		"Inductive Extractor"
+		"Remass Injector"
+		"XY-180 Radial Thrusters"
+		"Hyperdrive"
+
+	engine 0 75
+		gimbal 6
+	"reverse engine" -3 75
+		angle -30
+	"reverse engine" 3 75
+		angle 30
+	"steering engine" -36 -52
+		angle 90
+		right
+	"steering engine" 36 -52
+		angle -90
+		left
+	"steering engine" -3 75
+		angle 90
+		left
+	"steering engine" 3 75
+		angle -90
+		right
+
+	leak "leak" 30 90
+	leak "flame" 30 90
+	leak "big leak" 15 90
+	explode "tiny explosion" 18
+	explode "small explosion" 30
+	explode "medium explosion" 45
+	explode "large explosion" 12
+	"final explode" "final explosion medium"
+
+	description "The Frequentia-class Freighter is most commonly found serving as a fuel tender for ships in larger fleets, taking advantage of its large and ungainly cargo receptacle to store enough cryogenic deuterium to satisfy a small fleet's needs during long journeys. While its spherical fuel tank can be drained and used to transport dry goods and cargo, the cargo bay's notoriously small access door makes loading anything that can't be moved with a pump an exquisitely painful process."
+
+
+# Utility
+
+ship "Polari"
+	sprite "ship/avgi polari"
+	thumbnail "thumbnail/avgi polari"
+	attributes
+		category "Utility"
+		"cost" 2160000
+		"shields" 600
+		"hull"	2100
+		"required crew" 6
+		"bunks" 12
+		"mass" 240
+		"drag" 3.6
+		"heat dissipation" 0.9
+		"fuel capacity" 1800
+		"cargo space" 42
+		"outfit space" 294
+		"weapon capacity" 36
+		"engine capacity" 60
+		"asteroid scan power" 72
+		"anchor point" 1
+		weapon
+			"blast radius" 90
+			"shield damage" 600
+			"hull damage" 360
+			"hit force" 480
+	outfits
+		"Blue Optical Laser"
+		"Green Optical Laser"
```

Comment:
The Polari (and the Modified Tachytia, but that doesn't matter nearly as much) use more lasing power than their lasing generators allow, meaning if one is ever uninstalled it cannot be reinstalled. Did the Blue laser previously take up 2 lasing power instead?